docs/pages/docs/guides/choosing-a-database.mdx

```diff
@@ -0,0 +1,52 @@
+import { Markdown } from '../../../components/Markdown';
+import { Well } from '../../../components/primitives/Well';
+import { RelatedContent } from '../../../components/RelatedContent';
+
+# Choosing the right Database
+
+Keystone supports Postgres, MySQL and SQLite as database engines for your backend. There might be different reasons why you would choose one over the other.
```

Comment:
Should we link to something for `Postgres`, `MySQL`, `SQLite`?

Perhaps other parts of our documentation that show how to configure each? 🤷

Reply:
Added some links...